sources/packages/backend/apps/api/src/services/application-offering-change-request/application-offering-change-request.service.ts

```diff
@@ -156,15 +157,19 @@ export class ApplicationOfferingChangeRequestService {
 
   /**
    * Gets application offering request change list.
-   * @param locationId location id.
-   * @param paginationOptions options to execute the pagination.
    * @param statuses list of status that need to be included in the query.
+   * @param paginationOptions options to execute the pagination.
+   * @param options method options:
+   * - `locationId`: location for authorization.
    * @returns list of requested application offering changes.
    */
   async getSummaryByStatus(
-    locationId: number,
-    paginationOptions: PaginationOptions,
     statuses: ApplicationOfferingChangeRequestStatus[],
+    paginationOptions: PaginationOptions,
+    options?: {
+      locationId?: number;
+      useApplicationSort?: boolean;
```

Comment:
comment for useApplicationSort is missing.

Reply:
Added.